app/helpers/rubygems_helper.rb

```diff
@@ -54,36 +55,42 @@ def unsubscribe_link(rubygem)
             else
               'display:none'
             end
-    link_to t('.links.unsubscribe'), rubygem_subscription_path(rubygem),
+    link_to t('rubygems.show.links.unsubscribe'),
+      rubygem_subscription_path(rubygem),
```

Comment:
I've tried to extract performance issue and bug while working on this PR but this is my own bug from creating a `_aside.html.erb` partial in https://github.com/rubygems/rubygems.org/pull/945/commits/350485987f8247ef01c34ae899a3569952bd32cc
I will rebase and squash this fix to avoid misunderstanding